index.js

```diff
@@ -42,10 +43,17 @@ function onRequest () {
 		},
 	};
 }
+onRequest();
+`;
+
+const untrusted5 = `
```

Comment:
Pretend to be user, write your untrusted code, which will breach security of the VM2